samples/Sentry.Samples.Android/Sentry.Samples.Android.csproj

```diff
@@ -12,6 +12,9 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\src\Sentry.SourceGenerators\Sentry.SourceGenerators.csproj"
```

Comment:
Would SDK users need to reference this in order for the new functionality around detecting AOT etc. to work?

Reply:
No.  It gets built in via a None itemgroup in the Sentry.csproj.  I also had to add the targets file here manually in order to get the source gen to run in this project.  Users who get this via nuget don't have to worry about any of this